examples/cryptocurrency-advanced/backend/tests/api.rs

```diff
@@ -143,7 +146,7 @@ fn test_transfer_from_nonexisting_wallet() {
 fn test_transfer_to_nonexisting_wallet() {
     let (mut testkit, api) = create_testkit();
 
-    let (tx_alice, key_alice) = api.create_wallet(ALICE_NAME);
+    let (tx_alice, alice) = api.create_wallet(ALICE_NAME);
```

Comment:
Why though? IMO, it reads "Alice transfers _ to Bob".

Reply:
Yeah, it would've been perfectly fine, but Alice isn't transferring money here -- she creates a transaction, and before the transaction will be executed, Alice needs to send it as well (and reader can expect this action to be atomic). With `alice_wallet.transfer` it still a bit misleading, but not at that degree, as for me.